src/graph/ExecutionEngine.cpp

```diff
@@ -29,7 +29,11 @@ Status ExecutionEngine::init(std::shared_ptr<folly::IOThreadPoolExecutor> ioExec
         return addrs.status();
     }
     metaClient_ = std::make_unique<meta::MetaClient>(ioExecutor, std::move(addrs.value()));
-    metaClient_->init();
+    // load data try 3 time
+    bool loadDataOk = metaClient_->waitForMetadReady(3);
+    if (!loadDataOk) {
+        LOG(ERROR) << "ExecutionEngine::init loadData by thread error!";
```

Comment:
storaged return failed when load meta data failed, but why here not?

Reply:
when the data loading failed we should return ?

Reply:
Now, the code has changed!

Reply:
When load data error, then return error!